src/content/en/2021/compression.md

```diff
@@ -223,15 +234,17 @@ The HTTP Archive [runs Lighthouse audits](./methodology#lighthouse) for every mo
 
 ## How to improve on compression
 
-Before thinking about how to compress content, it is often wise to reduce the content to transmit to start with. One way of achieving this is to use so-called "minimizers", e.g. <a hreflang="en" href="https://github.com/kangax/html-minifier">HTMLMinifier</a>, <a hreflang="en" href="https://github.com/ben-eb/cssnano">CSSNano</a>, or <a hreflang="en" href="https://github.com/mishoo/UglifyJS2">UglifyJS</a>.
+Before thinking about how to compress content, it is often wise to reduce the content transmited to begin with. One way of achieving this is to use so-called "minimizers", such as <a hreflang="en" href="https://github.com/kangax/html-minifier">HTMLMinifier</a>, <a hreflang="en" href="https://github.com/ben-eb/cssnano">CSSNano</a>, or <a hreflang="en" href="https://github.com/mishoo/UglifyJS2">UglifyJS</a>.
 
-After having the minimal form of the content you want to transmit, the next step is to ensure compression is enabled. You can verify it is enabled as highlighted in the previous section, and configure your web server if needed.
+After having the minimal form of the conten to transmit, the next step is to ensure compression is enabled. You can verify it is enabled as highlighted in the previous section, and configure your web server if needed.
 
 When using only Gzip compression (also known as Deflate or Zlib), you can add support for Brotli. In comparison to Gzip, Brotli compresses to <a hreflang="en" href="https://quixdb.github.io/squash-benchmark/">smaller files at the same speed</a>, decompresses at the same speed, and is widely supported: <a hreflang="en" href="https://caniuse.com/brotli">can I use Brotli</a>.
 
+{# TODO - the wide support has been mentioned twice already #}
```

Comment:
That's a good point, the part "and is widely supported: <a hreflang="en" href="https://caniuse.com/brotli">can I use Brotli</a>." could be removed, especially since the brotli caniuse link is elsewhere as well

Reply:
Done, removed this here, and the other two mentions also got merged as far as I know thanks to an earlier TODO